src/doom/s_sound.c

```diff
@@ -764,6 +764,20 @@ void S_StartMusic(int m_id)
     S_ChangeMusic(m_id, false);
 }
 
+void S_PlayMusicLump(char *lumpname)
+{
+    musicinfo_t *music = NULL;
+    void *handle;
+
+    music = &S_music[0];
+    music->lumpnum = W_GetNumForName(lumpname);
```

Comment:
You should probably use `W_CheckLumpForName()` here and return gracefully if no lump of that name could be found.